transformations/numeric_to_word/abbreviated_currency_symbols_to_currency_name_dict.py

```diff
@@ -0,0 +1,151 @@
+abbreviated_currency_symbols_to_currency_name_dict = {'AFN': 'Afghanistani Afghani',
```

Comment:
Better to have one file for constants, it would be great if we can put both 'abbreviated_currency_symbols_to_currency_name_dict' and 'symbol_to_currency_name_dict' in single file.

Reply:
Yes you're correct. My bad on those. I've put both constants in single file, `constants.py`. Thanks for the feedback @aadesh11!